stdlib/public/Concurrency/Task.swift

```diff
@@ -596,14 +609,22 @@ public func withUnsafeCurrentTask<T>(body: (UnsafeCurrentTask?) throws -> T) ret
   return try body(UnsafeCurrentTask(_task))
 }
 
-/// An *unsafe* 'current' task handle.
+/// An unsafe task handle for the current task.
 ///
-/// An `UnsafeCurrentTask` should not be stored for "later" access.
+/// To get an instance of `UnsafeCurrentTask` for the current task,
+/// call the `withUnsafeCurrentTask(body:)` method.
+/// Don't try to store an unsafe task handle
+/// for use outside that method's closure.
+/// Storing an unsafe task handle has no impact on the task's actual lifecycle,
+/// and the behavior of accessing an unsafe task handle
+/// outside of the `withUnsafeCurrentTask(body:)` method's closure isn't defined.
+/// Instead, use the `task` property of `UnsafeCurrentTask`
+/// to access an instance of `Task` that you can store long-term
+/// and interact with outside of the closure body.
 ///
-/// Storing an `UnsafeCurrentTask` has no implication on the task's actual lifecycle.
-///
-/// The sub-set of APIs of `UnsafeCurrentTask` which also exist on `Task` are
+/// The subset of APIs of `UnsafeCurrentTask` which also exist on `Task` are
 /// generally safe to be invoked from any task/thread.
+/// ◊TR: Which ones, exactly, is that?  "Generally" isn't good enough here.
```

Comment:
we can remove the "generally" the trick is "if an API exists on Task, it is also safe on UnsafeCurrentTask from other contexts; If it only exists on UnsafeCurrentTask, it is unsafe to call it from other tasks"